stdlib/universe/table_fns.go

```diff
@@ -88,6 +88,11 @@ func tableFindCall(ctx context.Context, args values.Object) (values.Value, error
 	if err != nil {
 		return nil, errors.Wrap(err, codes.Inherit, "error in table object compilation")
 	}
+
+	if !lang.HaveExecutionDependencies(ctx) {
```

Comment:
Unless I'm missing something, `err` is `nil` here, right?  So there is no error to be wrapped or code to inherit?

Reply:
Another note on this, the error code should be `codes.Invalid` to reflect that the user did something unsupported, since if it is `codes.Internal`, we will hide the message from the user as a security measure.

Reply:
Thank you for the excellent suggestions! Pushed a new version with a more detailed error message, no error wrapping, using Invalid error type and with a test case.